plugins/modules/eos_pyateos.py

```diff
@@ -0,0 +1,568 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2020 Red Hat
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "network",
+}
+
+DOCUMENTATION = """
+---
+module: eos_pyateos
+version_added: "2.10"
+author:
+  - "Federico Olivieri (@Federico87)"
+
+short_description: Operational status tests on Arista device.
+description: A snapshot of the operational status of a switch is taken before a
+    config or network change and compare against a second snapshot taken after the change.
+    A diff file is generated in .json format.
+extends_documentation_fragment: eos
```

Comment:
```suggestion
```

All this does is define the `provider` argument (and by extension `connection: local`), which we are not supporting for new modules. You should also likewise remove `provider: '{{ eapi }}'` from your tests below, as even if you were intending to support it, there is no actual testing using `connection: local`

Reply:
Ok. I just reverse engineering a test module from collection. I was not 100% sure how the session was handled so I decided to keep like the example I was following